compiler/optimizer/StructuralAnalysis.hpp

```diff
@@ -92,15 +92,15 @@ class TR_RegionAnalysis
 
    TR_RegionAnalysis(TR::Compilation *comp, TR_Dominators &dominators, TR::CFG * cfg, TR::Region &workingRegion) :
       _workingRegion(workingRegion),
```

Comment:
Should this also be renamed in this PR, or is it slated to be cleaned up in a future PR?

Reply:
Sure, I'll do that one in this PR. Thanks for spotting that.

Reply:
Fixed in force push.